src/pytest_mock_resources/container/__init__.py

```diff
@@ -67,3 +67,4 @@ def retriable_check_fn(retries):
 
 from pytest_mock_resources.container.mongo import _mongo_container  # noqa, isort:skip
```

Comment:
can you add a `# flake8: noqa` to the top, and remove them from each individual line? Also remove isort:skip, that should be unnecessary